.codeclimate.yml

```diff
@@ -1,4 +1,12 @@
 engines:
+  golint:
```

Comment:
What's the purpose of disabling specific paths if we don't use golint? Couldn't we just flag it as disabled?

Reply:
@sgnn7 Sorry I don't follow what you're asking.  Can you clarify?